projects/example-app/src/app/books/actions/collection-api.actions.ts

```diff
@@ -0,0 +1,64 @@
+import { Action } from '@ngrx/store';
+import { Book } from '../models/book';
+
+export enum CollectionApiActionTypes {
+  AddBookSuccess = '[Collection/API] Add Book Success',
+  AddBookFail = '[Collection/API] Add Book Fail',
+  RemoveBookSuccess = '[Collection/API] Remove Book Success',
+  RemoveBookFail = '[Collection/API] Remove Book Fail',
+  LoadSuccess = '[Collection/API] Load Success',
```

Comment:
Since the other actions all have the word `Book` in the enum and in the value, perhaps it would be better to use `LoadBooksSuccess = [Collection/API] Load Books Success`?

Reply:
I agree.